### PR TITLE
Makes borg modules and human species easily spawnable by admins from the mob spawn menu.

### PR DESCRIPTION
**Pull request by Firecage**
This makes it so admins can easily spawn the various borg modules and human species from the mob spawn menu at the game panel. Tested and everything seems to work. Remember to equip your plasmamen after spawning them.

_Cyborg Service Modules not included in this product due to issues I can't figure out yet how to resolve. Please buy this product seperately._

### CODE REVIEW

**Comment by GunHog** [comment]
Wonderful. This adds support for a PR I want to do where a Traitor AI can force a module selection on borgs created with the factory!

**Comment by kevinz000** [comment]
damnit firecage it's a rite of passage to know how to abuse this stuff without code!!

**Comment by Firecage** [comment]
Though yah, the Service Borg module has an issue. Normally when people select it, you get a pop up to choose your sprite, and then the module is activated. When you directly try to spawn a service borg, well, it just makes a default borg. So will have to figure out how to solve that. Perhaps have them become the service borg module first, then have to use a button to select their sprite afterwards?

**Comment by GunHog** [comment]
@Firecage that works, or have it such that the module proc takes a forced argument. If it is forced, the module will randomly select a sprite for the borg. I prefer you solution, though.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -781,7 +781,35 @@
 		cell = null
 	qdel(src)
 
-/mob/living/silicon/robot/syndicate
+/mob/living/silicon/robot/modules
+	var/set_module = null
+
+/mob/living/silicon/robot/modules/Initialize()
+	. = ..()
+	module.transform_to(set_module)
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/robot/robot.dm]: Don't make a duplicate definition

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**